test/unit/StakingPool/getPrices.js

```diff
@@ -0,0 +1,119 @@
+const { assert } = require('chai');
+const { web3, ethers: { utils: { parseEther } } } = require('hardhat');
+const { time, expectRevert } = require('@openzeppelin/test-helpers');
```

Comment:
Use waffle for reverts. Avoid oz test-helpers since it is something which we're supposed to remove.

https://ethereum-waffle.readthedocs.io/en/latest/matchers.html?highlight=revert#revert-with-message